openedx/features/enterprise_support/api.py

```diff
@@ -301,6 +304,32 @@ def get_enterprise_customer(self, uuid):
         return enterprise_customer
 
 
+def activate_learner_enterprise(request, user, enterprise_customer):
```

Comment:
Can we please move this code to the [edx-enterprise](https://github.com/edx/edx-enterprise) repo if it is not possible to use this [method](https://github.com/edx/edx-enterprise/blob/69ca5a6eb947c19f8a8764145c3c42695590772a/enterprise/views.py#L1095). Since we have all the code related to Multiple Enterprise in the edx-enterprise repo.

Reply:
Our initial plan was to just post call the view you mentioned in your comment. Should we do that instead?

Reply:
Can you please share more on it, the reason why the plan was changed afterwords? I think by calling that view there is less work here and also we will be using the same code.

Reply:
I noticed that `api.py` was used for enterprise related work and also I am calling `get_enterprise_learner_data_from_api` from there as well. I can move to calling the view directly